proto/tikvpb.proto

```diff
@@ -61,6 +61,7 @@ service Tikv {
     // Region commands.
     rpc SplitRegion (kvrpcpb.SplitRegionRequest) returns (kvrpcpb.SplitRegionResponse) {}
     rpc ReadIndex(kvrpcpb.ReadIndexRequest) returns (kvrpcpb.ReadIndexResponse) {}
+    rpc GetCommittedIndexAndTs(kvrpcpb.GetCommittedIndexAndTsRequest) returns (kvrpcpb.GetCommittedIndexAndTsResponse) {}
```

Comment:
Without modify `BatchCommandsRequest` you can't use it `BatchCommands`. So I prefer to put it in an new service to make development faster.